Dockerfile.compute-node

```diff
@@ -579,6 +579,26 @@ RUN wget https://gitlab.com/dalibo/postgresql_anonymizer/-/archive/1.1.0/postgre
     sort -o /before.txt /before.txt && sort -o /after.txt /after.txt && \
     comm -13 /before.txt /after.txt | tar --directory=/usr/local/pgsql --zstd -cf /extensions/anon.tar.zst -T -
 
+#########################################################################################
```

Comment:
+ @bayandin 

Can we compile some extensions privately? I don't think that we can put them publicly for some of our customers.

Reply:
We can set up a CI workflow in a private repo and call it from the neon repo (the same we do it for e2e tests).

Reply:
I like this idea, BTW. @awestover, can you make it real? :)

Reply:
I'll try

Reply:
awesome!

Reply:
oh actually right now it is flipped: the workflow gets skipped in remote repo when run from here, but gets run if run from there. need to unflip it